src/cli/providers/messenger/persona.js

```diff
@@ -211,7 +211,7 @@ export async function deletePersona(ctx) {
       print(`Cannot get persona of ID ${bold(personaId)}`);
     }
   } catch (err) {
-    error(`Failed to create ${bold('persona')}`);
+    error(`Failed to delete ${bold('persona')} ${bold(personaId)}`);
```

Comment:
Here should add `of ID` for consistency

Reply:
I changed both of them to `with ID`.

Reply:
I think you specify the ID of the persona to delete. So using `of` is still better in my opinion.